packages/compass-e2e-tests/tests/search-indexes.test.ts

```diff
@@ -114,7 +119,10 @@ async function dropSearchIndex(browser: CompassBrowser, indexName: string) {
   await modal.waitForDisplayed({ reverse: true });
 
   await browser.waitUntil(async () => {
-    return await indexRow.waitForExist({ reverse: true });
+    return await indexRow.waitForExist({
```

Comment:
I think the waitUntil wrapping this is superfluous? I get that that was already the case before. But also the waitUntil's using the default timeout and that's shorter than this one used by the waitForExist.

Reply:
Yep, that makes sense. Will remove it